spec/spec_helper.rb

```diff
@@ -18,4 +18,9 @@ def format(result)
 RSpec.configure do |config|
   config.filter_run :focus => true
   config.run_all_when_everything_filtered = true
+
+  config.before do
+    # allow(Adhearsion).to receive(:new_request_id).and_return SecureRandom.uuid # adhearsion:adhearsion-3
+    allow(Adhearsion).to receive(:new_request_id).and_return('12345678-aaaa-bbbb-cccc-123456789012') # kares:ahn3
```

Comment:
@gfaza which one of these do you think we should go for?

Reply:
tl;dr:  adhearsion's version well documents what type of response we should expect, whereas kares's version is more performant.  let's meet them both in the middle by using a constant that illustrates the nature of the response we should expect, while minimizing the number of calculations.

i prefer mocking to occur closer to (i.e., within) the spec.  i'd favor having a helper function here along the lines of 
```
def mock_new_request_id(response)
  response ||= SecureRandom.uuid
  allow(Adhearsion).to receive(:new_request_id).and_return(SecureRandom.uuid)
end
```
and call that method as needed from within specs.  however, this is moot b/c none of the specs are actually depending on the return value - they only care that the call to `Adhearsion#new_request_id` gets stubbed such that related Adhearsion internals aren't exercised.

Reply:
I completely agree, nice approach, man 👏   I dig the constant and it well-exhibits the expected type of output.